app/Projects/Applications/Templates/Html.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+namespace Servidor\Projects\Applications\Templates;
+
+use Servidor\Projects\Application;
+
+class Html
+{
+    /** @var Application */
+    protected $app;
+
+    /** @var string */
+    public $nginxTemplate = 'basic';
+
+    /** @var string */
+    public $publicDir = '';
+
+    public function __construct(Application $app)
+    {
+        $this->app = $app;
+    }
+
+    public function disable(): void
+    {
+        $config = $this->app->domain_name . '.conf';
+        $symlink = '/etc/nginx/sites-enabled/' . $config;
+
+        if (!is_link($symlink) || readlink($symlink) !== '/etc/nginx/sites-available/' . $config) {
+            return;
+        }
+
+        exec('sudo rm "' . $symlink . '"');
```

Comment:
Seems to be failing to remove symlink on travis, but works locally. Joy.

```suggestion
        $config = $this->app->domain_name . '.conf';
        $symlink = '/etc/nginx/sites-enabled/' . $config;
        var_dump("Request to remove symlink...", $config, $symlink);
        var_dump("Is link?", is_link($symlink));
        var_dump("Matches?", readlink($symlink) === '/etc/nginx/sites-available/' . $config);

        if (!is_link($symlink) || readlink($symlink) !== '/etc/nginx/sites-available/' . $config) {
            var_dump("Bailing out! Not a link, or link was changed");

            return;
        }

        var_dump("Attempting removal!");
        exec('sudo rm "' . $symlink . '"', $output, $exit);
        echo "Got exit status ${exit} with this output: " . print_r($output, true);
```